packages/bundler/src/runBundler.ts

```diff
@@ -166,6 +177,13 @@ export async function runBundler (argv: string[], overrideExit = true): Promise<
   const {
     entryPoint
   } = await connectContracts(wallet, !config.rip7560)
+
+  if (entryPoint != null && entryPoint?.address !== config.entryPoint && [1337, 31337].includes(chainId)) {
+    console.log('NOTICE: overriding config entrypoint: ', { entryPoint: entryPoint.address })
```

Comment:
Please `console.warn` it - seems like something you don't actually want most of the time.